src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -160,6 +161,9 @@ export interface IPlotsTabProps {
     store:ResultsViewPageStore;
 };
 
+export type Option = {value:string, label:string};
+export type DataSource = {[dataType:string]:{value:string, label:string}[]};
```

Comment:
ditto here.  if it's exported and not meant for general use, maybe give it a more specific name

Reply:
Renamed to `PlotsTabDataSource`